public/static/docs/command-reference/update.md

```diff
@@ -69,3 +81,41 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
+[get started example repo](https://github.com/iterative/example-get-started) at
+first:
+
+```dvc
+$ dvc import --rev baseline-experiment git@github.com:iterative/example-get-started model.pkl
+Importing 'model.pkl (git@github.com:iterative/example-get-started)'
+-> 'model.pkl'
+```
+
+After this, the import stage (DVC-file) `model.pkl.dvc` is created.
+
+Let's try to run `dvc update` on the given stage file, and see what happens.
+
+```dvc
+$ dvc update model.pkl.dvc
+```
+
+There was no output at all, meaning, the `model.pkl` file was not updated. This
+is because, we tied the import stage with a `rev` that never changes (i.e. tag
+is tied to a specific commit). Therefore, it was not updated.
```

Comment:
```suggestion
is because we tied the import stage to a `rev` that hasn't changed (i.e.
`baseline-experiment` tag points to a specific Git commit). Therefore, it was not updated.
```
But again, this may be better near the top to explain what a fixed revision is... or just copy the example from `dvc import` instead.